<!doctype html>

<link href="css/style.css" rel="stylesheet">
<head>
	<meta charset="utf-8">
        
	<title>Systems and Population Genetics Group</title>
	<meta name="description" content="">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" href="assets/css/bootstrap.min.css">
        <link rel="stylesheet" href="assets/css/plugins.css" />
        <link rel="stylesheet" href="assets/css/lora-web-font.css" />
        <link rel="stylesheet" href="assets/css/opensans-web-font.css" />
        <link rel="stylesheet" href="assets/css/magnific-popup.css">
        <link rel="stylesheet" href="assets/css/style.css">
        <link rel="stylesheet" href="assets/css/responsive.css" />
</head>

<body>
	<nav class="navbar navbar-default navbar-fixed-top" id="main_navbar">
		<div class="container">
        	        <div class="navbar-header">
        	            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        	                <span class="sr-only">Toggle navigation
        	                <span class="icon-bar">
        	                <span class="icon-bar">
        	                <span class="icon-bar">
        	            </button>
        	        </div>

        	        <!-- Collect the nav links, forms, and other content for toggling -->
			<!-- <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"> -->
			<div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
        	            <ul class="nav navbar-nav navbar-right">
        	                <li><a href="#home">Home</a></li>
        	                <li><a href="#activities">Activities</a></li>
        	                <li><a href="#software">Software</a></li>
        	                <li><a href="#members">Group members</a></li>
        	                <li><a href="#publications">Publications</a></li>
        	                <li><a href="#contact">Contact</a></li>
        	            </ul>
        	        </div><!-- /.navbar-collapse -->
        	    </div><!-- /.container-fluid -->
	</nav>

        <!--Home page style-->
        <header id="home" class="home">
            <div class="overlay">
                <div class="container">
                    <div class="row">
                        <div class="home-wrapper">
                            <div class="col-md-12 col-sm-12 col-xs-12">
                                <div class="home-content text-center">
                                    <h1>Systems and Population Genetics Group</h1>
					<h3>Department of Computational Biology</h3>
					<h3>University of Lausanne</h3>
					
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </header>

        <!-- Sections -->
        <section id="activities" class="sections">
            <div class="container">
                <div class="heading text-center">
                    <h1>Activities</h1>
                    <div class="separator"></div>
                </div>
                <!-- Example row of columns -->
                <div class="row">
                    <div class="wrapper">
                        <div class="col-md-4 col-sm-6 col-xs-12">
                            <div class="service-item text-center">
                                <i class="fa fa-code-fork"></i>
                                <h5>Systems genetics</h5>
                                <div class="separator2"></div>
                                <p>We are interested by the regulatory machinery controlling gene expression. We investigate it by analysing population scale multi-omics datasets (e.g. genomics, transcriptomics, epigenomics, proteomics) in which we use genetic variations as natural perturbations of the systems in order to unravel the interactions between molecular features (e.g. genes, regulatory elements, proteins).</p>
                            </div>
                        </div> 

                        <div class="col-md-4 col-sm-6 col-xs-12">
                            <div class="service-item text-center">
                                <i class="fa fa-male"></i>
                                <h5>Population genetics</h5>
                                <div class="separator2"></div>
                                <p>We are also interested by characterizing how individuals are genetically related in a population. We look at this by identifying shared chromosome stretches (i.e. haplotypes) between individuals in a given population. To do so, we apply sophisticated statistical models in large scale genomics datasets containing up to millions of genetic variations typed and thousands of individuals.</p>
                            </div>
                        </div>

                        <div class="col-md-4 col-sm-6 col-xs-12">
                            <div class="service-item text-center">
                                <i class="fa fa-terminal"></i>
                                <h5>Method Development</h5>
                                <div class="separator2"></div>
                                <p>One leitmotiv in our research is the development of innovative computational methods to efficiently process and analyse large scale genomic datasets. We maintain multiple software that have been developed in previous work and that have been used in collaborative projects such as the <a href="http://www.internationalgenome.org/">1000 Genomes project</a>, <a href="http://www.ukbiobank.ac.uk/">UK biobank</a> and <a href="https://www.gtexportal.org/home/">GTEx</a>.</p>
                            </div>
                        </div> 

                    </div>
                </div>
            </div> <!-- /container -->       
        </section>

        <!-- Sections -->
        <section id="software" class="lightbg sections">
            <div class="container">
                <div class="heading text-center">
                    <h1>Software</h1>
                    <div class="separator"></div>
                </div>
                <!-- Example row of columns -->
                <div class="row">
                    <div class="wrapper">
			<div class="col-md-3 col-sm-4 col-xs-12">
			    <div class="service-item text-center">
                                <h5>QTLtools</h5>
                                <div class="separator2"></div>
				<p><a href="https://qtltools.github.io/qtltools/">LINK</a></p>
                                <p><b>QTLtools</b> is a tool set for molecular QTL discovery and analysis.
				It allows to go from the raw sequence data to collection of molecular Quantitative Trait Loci (QTLs) in few easy-to-perform steps.</p>
                            </div>
                        </div> 

                        <div class="col-md-3 col-sm-4 col-xs-12">
                            <div class="service-item text-center">
                                <h5>FastQTL</h5>
                                <div class="separator2"></div>
				<p><a href="https://github.com/francois-a/fastqtl">LINK</a></p>
                                <p><b>FastQTL</b> is a fast QTL mapper for molecular phenotypes such as gene expression measured via RNA-seq. Used to map cis-eQTL in <a href="https://www.gtexportal.org/home/">GTEx</a>.</p>
                            </div>
                        </div>

                        <div class="col-md-3 col-sm-4 col-xs-12">
                            <div class="service-item text-center">
                                <h5>SHAPEIT</h5>
                                <div class="separator2"></div>
				<p><a href="https://odelaneau.github.io/shapeit4/">LINK</a></p>
                                <p><b>SHAPEIT</b> is a fast and accurate method for estimation of haplotypes (aka phasing) from genotype or sequencing data. Used to infer haplotypes in <a href="http://www.internationalgenome.org/">1000 Genomes</a>, <a href="http://www.ukbiobank.ac.uk/">UK biobank</a> and <a href="http://www.haplotype-reference-consortium.org/">HRC</a>.</p>
				             </div>
                        </div> 

                        <div class="col-md-3 col-sm-4 col-xs-12">
                            <div class="service-item text-center">
                                <h5>GLIMPSE</h5>
                                <div class="separator2"></div>
				<p><a href="https://odelaneau.github.io/GLIMPSE/">LINK</a></p>
                                <p><b>GLIMPSE</b> is a fast and accurate method for imputing haplotypes and genotypes from low-coverage sequencing data.</p>
				             </div>
                        </div> 


                    </div>
                </div>
            </div> <!-- /container -->       
        </section>

        <!-- Sections -->
        <section id="members" class="sections">
        	<div class="container">
        		<div class="heading text-center">
                		<h1>Group Members</h1>
                		<div class="separator"></div>
                	</div>
                	
                	<!-- Example row of columns -->
                	<div class="row">
                		<div class="wrapper">
                			<div class="col-md-3 col-sm-4 col-xs-12">
			    			<div class="service-item text-center">
							<img src="assets/images/Olivier.jpg" alt="Olivier Delaneau">
							<h5>Olivier<br>Delaneau</h5>
		                	                <div class="separator2"></div>
                			                <p>SNSF professor</p>
                				</div>
                        		</div>

                        		<div class="col-md-3 col-sm-4 col-xs-12">
						<div class="service-item text-center">
							<img src="assets/images/Diogo.jpg" alt="Diogo Ribeiro">
			                                <h5>Diogo<br>Ribeiro</h5>
        	        		                <div class="separator2"></div>
        	        		                <p>Postdoc</p>
        	        		            </div>
					</div>
			
        		                <div class="col-md-3 col-sm-4 col-xs-12">
			    			<div class="service-item text-center">
							<img src="assets/images/Simone.png" alt="Simone Rubinacci">
        			                        <h5>Simone<br>Rubinacci</h5>
        			                        <div class="separator2"></div>
        			                        <p>Postdoc</p>
						</div>
					</div>

					<div class="col-md-3 col-sm-4 col-xs-12">
						<div class="service-item text-center">
							<img src="assets/images/Robin.jpg" alt="Robin Hofmeister">
                					<h5>Robin<br>Hofmeister</h5>
                					<div class="separator2"></div>
		                    			<p>PhD student</p>
                				</div>
            				</div>
            			

					<div class="col-md-3 col-sm-4 col-xs-12">
						<div class="service-item text-center">
							<img src="assets/images/Diana.png" alt="Robin Hofmeister">
                					<h5>Diana<br>Avalos</h5>
                					<div class="separator2"></div>
		                    			<p>PhD student</p>
                				</div>
            				</div>
            			
					<div class="col-md-3 col-sm-4 col-xs-12">
						<div class="service-item text-center">
							<img src="assets/images/Barbara.jpg" alt="Barbara Mota">
                					<h5>Barbara<br>Mota</h5>
                					<div class="separator2"></div>
	        	            			<p>PhD student</p>
        		       			</div>
            				</div>
		
					<div class="col-md-3 col-sm-4 col-xs-12">
						<div class="service-item text-center">
							<img src="assets/images/Rick.png" alt="Barbara Mota">
                					<h5>Rick<br>Wertenbroek</h5>
                					<div class="separator2"></div>
	        	            			<p>PhD student</p>
        		       			</div>
            				</div>
                    		</div>
	                </div>
        	</div> <!-- /container -->       
        </section>

        <section id="publications" class="lightbg sections">
            <div class="container">
                <div class="heading text-center">
                    <h1>Publications (copy/paste from PUBMED)</h1>
                    <p><a href="https://scholar.google.ch/citations?user=lpg23rgAAAAJ&hl=en">Google Scholar</a></p>
                    <div class="separator"></div>
                </div>

<!--    PUBLICATIONS vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
	<table style="width:100%">
	
	
	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/36042219/" target="_blank">Deconvoluting complex correlates of COVID-19 severity with a multi-omic pandemic tracking strategy.</a></p>
			<p>Parikh VN, Ioannidis AG, Jimenez-Morales D, Gorzynski JE, De Jong HN, Liu X, Roque J, Cepeda-Espinoza VP, Osoegawa K, Hughes C, Sutton SC, Youlton N, Joshi R, Amar D, Tanigawa Y, Russo D, Wong J, Lauzon JT, Edelson J, Mas Montserrat D, Kwon Y, Rubinacci S, Delaneau O, Cappello L, Kim J, Shoura MJ, Raja AN, Watson N, Hammond N, Spiteri E, Mallempati KC, Montero-Martín G, Christle J, Kim J, Kirillova A, Seo K, Huang Y, Zhao C, Moreno-Grau S, Hershman SG, Dalton KP, Zhen J, Kamm J, Bhatt KD, Isakova A, Morri M, Ranganath T, Blish CA, Rogers AJ, Nadeau K, Yang S, Blomkalns A, O'Hara R, Neff NF, DeBoever C, Szalma S, Wheeler MT, Gates CM, Farh K, Schroth GP, Febbo P, deSouza F, Cornejo OE, Fernandez-Vina M, Kistler A, Palacios JA, Pinsky BA, Bustamante CD, Rivas MA, Ashley EA.</p>
			<p>Nat Commun. 2022 Aug 30;13(1):5107. doi: 10.1038/s41467-022-32397-8.</p>
			<p>PMID: 36042219</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/36028576/" target="_blank">Shared regulation and functional relevance of local gene co-expression revealed by single cell analysis.</a></p>
			<p>M Ribeiro D, Ziyani C, Delaneau O.</p>
			<p>Commun Biol. 2022 Aug 26;5(1):876. doi: 10.1038/s42003-022-03831-w.</p>
			<p>PMID: 36028576</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/35748697/" target="_blank">XSI - A genotype compression tool for compressive genomics in large biobanks.</a></p>
			<p>Wertenbroek R, Rubinacci S, Xenarios I, Thoma Y, Delaneau O.</p>
			<p>Bioinformatics. 2022 Jun 24;38(15):3778-84. doi: 10.1093/bioinformatics/btac413. Online ahead of print.</p>
			<p>PMID: 35748697</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/35176773/" target="_blank">GCAT|Panel, a comprehensive structural variant haplotype map of the Iberian population from high-coverage whole-genome sequencing.</a></p>
			<p>Valls-Margarit J, Galván-Femenía I, Matías-Sánchez D, Blay N, Puiggròs M, Carreras A, Salvoro C, Cortés B, Amela R, Farre X, Lerga-Jaso J, Puig M, Sánchez-Herrero JF, Moreno V, Perucho M, Sumoy L, Armengol L, Delaneau O, Cáceres M, de Cid R, Torrents D.</p>
			<p>Nucleic Acids Res. 2022 Mar 21;50(5):2464-2479. doi: 10.1093/nar/gkac076.</p>
			<p>PMID: 35176773</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/34561431/" target="_blank">Differentially expressed genes reflect disease-induced rather than disease-causing changes in the transcriptome.</a></p>
			<p>Porcu E, Sadler MC, Lepik K, Auwerx C, Wood AR, Weihs A, Sleiman MSB, Ribeiro DM, Bandinelli S, Tanaka T, Nauck M, Völker U, Delaneau O, Metspalu A, Teumer A, Frayling T, Santoni FA, Reymond A, Kutalik Z.</p>
			<p>Nat Commun. 2021 Sep 24;12(1):5647. doi: 10.1038/s41467-021-25805-y.</p>
			<p>PMID: 34561431</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/34376650/" target="_blank">The molecular basis, genetic control and pleiotropic effects of local gene co-expression.</a></p>
			<p>Ribeiro DM, Rubinacci S, Ramisch A, Hofmeister RJ, Dermitzakis ET, Delaneau O.</p>
			<p>Nat Commun. 2021 Aug 10;12(1):4842. doi: 10.1038/s41467-021-25129-x.</p>
			<p>PMID: 34376650</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/34237774/" target="_blank">Mapping the human genetic architecture of COVID-19.</a></p>
			<p>COVID-19 Host Genetics Initiative.</p>
			<p>Nature. 2021 Dec;600(7889):472-477. doi: 10.1038/s41586-021-03767-x. Epub 2021 Jul 8.</p>
			<p>PMID: 34237774</p>
		</td>
	</tr>

	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/33930288/" target="_blank">The genomic history of the Aegean palatial civilizations.</a></p>
			<p>Clemente F, Unterländer M, Dolgova O, Amorim CEG, Coroado-Santos F, Neuenschwander S, Ganiatsou E, Cruz Dávalos DI, Anchieri L, Michaud F, Winkelbach L, Blöcher J, Arizmendi Cárdenas YO, Sousa da Mota B, Kalliga E, Souleles A, Kontopoulos I, Karamitrou-Mentessidi G, Philaniotou O, Sampson A, Theodorou D, Tsipopoulou M, Akamatis I, Halstead P, Kotsakis K, Urem-Kotsou D, Panagiotopoulos D, Ziota C, Triantaphyllou S, Delaneau O, Jensen JD, Moreno-Mayar JV, Burger J, Sousa VC, Lao O, Malaspinas AS, Papageorgopoulou C.</p>
			<p>Cell. 2021 May 13;184(10):2565-2586.e21. doi: 10.1016/j.cell.2021.03.039. Epub 2021 Apr 29.</p>
			<p>PMID: 33930288</p>
		</td>
	</tr>
	
	<tr>
		<td>	<p><a href="https://pubmed.ncbi.nlm.nih.gov/33542415/" target="_blank">Gene regulation contributes to explain the impact of early life socioeconomic disadvantage on adult inflammatory levels in two cohort studies. </a></p>
			<p>Carmeli C, Kutalik Z, Mishra PP, Porcu E, Delpierre C, Delaneau O, Kelly-Irving M, Bochud M, Dhayat NA, Ponte B, Pruijm M, Ehret G, Kähönen M, Lehtimäki T, Raitakari OT, Vineis P, Kivimäki M, Chadeau-Hyam M, Dermitzakis E, Vuilleumier N, Stringhini S.</p>
			<p>Sci Rep. 2021 Feb 4;11(1):3100. doi: 10.1038/s41598-021-82714-2.</p>
			<p>PMID: 33542415 Free PMC article. </p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/33414550/" target="_blank">Efficient
			phasing and imputation of low-coverage sequencing data using large
			reference panels. </a>
			</p>
			<p>Rubinacci
			S, Ribeiro DM, Hofmeister RJ, Delaneau O.</p>
			<p>Nat
			Genet. 2021 Jan;53(1):120-126. doi: 10.1038/s41588-020-00756-0.
			Epub 2021 Jan 7.</p>
			<p>PMID:
			33414550 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/33196638/" target="_blank">Genotype
			imputation using the Positional Burrows Wheeler Transform. </a>
			</p>
			<p>Rubinacci
			S, Delaneau O, Marchini J.</p>
			<p>PLoS
			Genet. 2020 Nov 16;16(11):e1009049. doi:
			10.1371/journal.pgen.1009049. eCollection 2020 Nov.</p>
			<p>PMID:
			33196638 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/32958763/" target="_blank">Retrospective
			evaluation of whole exome and genome mutation calls in 746 cancer
			samples. </a>
			</p>
			<p>Bailey
			MH, Meyerson WU, Dursi LJ, Wang LB, Dong G, Liang WW, Weerasinghe
			A, Li S, Li Y, Kelso S; MC3 Working Group; PCAWG novel somatic
			mutation calling methods working group, Saksena G, Ellrott K,
			Wendl MC, Wheeler DA, Getz G, Simpson JT, Gerstein MB, Ding L;
			PCAWG Consortium.</p>
			<p>Nat
			Commun. 2020 Sep 21;11(1):4748. doi: 10.1038/s41467-020-18151-y.</p>
			<p>PMID:
			32958763 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/32859912/" target="_blank">Sex
			differences in oncogenic mutational processes. </a>
			</p>
			<p>Li
			CH, Prokopec SD, Sun RX, Yousif F, Schmitz N; PCAWG Tumour
			Subtypes and Clinical Translation, Boutros PC; PCAWG Consortium.</p>
			<p>Nat
			Commun. 2020 Aug 28;11(1):4330. doi: 10.1038/s41467-020-17359-2.</p>
			<p>PMID:
			32859912 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/32766602/" target="_blank">High-throughput
			SARS-CoV-2 and host genome sequencing from single nasopharyngeal
			swabs. </a>
			</p>
			<p>Gorzynski
			JE, De Jong HN, Amar D, Hughes CR, Ioannidis A, Bierman R, Liu D,
			Tanigawa Y, Kistler A, Kamm J, Kim J, Cappello L, Neff NF,
			Rubinacci S, Delaneau O, Shoura MJ, Seo K, Kirillova A, Raja A,
			Sutton S, Huang C, Sahoo MK, Mallempati KC, Montero-Martin G,
			Osoegawa K, Jimenez-Morales D, Watson N, Hammond N, Joshi R,
			Fernandez-Vina M, Christle JW, Wheeler MT, Febbo P, Farh K,
			Schroth G, Desouza F, Palacios J, Salzman J, Pinsky BA, Rivas MA,
			Bustamante CD, Ashley EA, Parikh VN.</p>
			<p>medRxiv.
			2020 Jul 29:2020.07.27.20163147. doi: 10.1101/2020.07.27.20163147.
			Preprint.</p>
			<p>PMID:
			32766602 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/32025007/" target="_blank">Pan-cancer
			analysis of whole genomes. </a>
			</p>
			<p>ICGC/TCGA
			Pan-Cancer Analysis of Whole Genomes Consortium.</p>
			<p>Nature.
			2020 Feb;578(7793):82-93. doi: 10.1038/s41586-020-1969-6. Epub
			2020 Feb 5.</p>
			<p>PMID:
			32025007 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/31780650/" target="_blank">Accurate,
			scalable and integrative haplotype estimation. </a>
			</p>
			<p>Delaneau
			O, Zagury JF, Robinson MR, Marchini JL, Dermitzakis ET.</p>
			<p>Nat
			Commun. 2019 Nov 28;10(1):5436. doi: 10.1038/s41467-019-13225-y.</p>
			<p>PMID:
			31780650 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/31048460/" target="_blank">Chromatin
			three-dimensional interactions mediate genetic effects on gene
			expression. </a>
			</p>
			<p>Delaneau
			O, Zazhytska M, Borel C, Giannuzzi G, Rey G, Howald C, Kumar S,
			Ongen H, Popadin K, Marbach D, Ambrosini G, Bielser D, Hacker D,
			Romano L, Ribaux P, Wiederkehr M, Falconnet E, Bucher P, Bergmann
			S, Antonarakis SE, Reymond A, Dermitzakis ET.</p>
			<p>Science.
			2019 May 3;364(6439):eaat8266. doi: 10.1126/science.aat8266.</p>
			<p>PMID:
			31048460 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/31009447/" target="_blank">Expression
			estimation and eQTL mapping for HLA genes with a personalized
			pipeline. </a>
			</p>
			<p>Aguiar
			VRC, César J, Delaneau O, Dermitzakis ET, Meyer D.</p>
			<p>PLoS
			Genet. 2019 Apr 22;15(4):e1008091. doi:
			10.1371/journal.pgen.1008091. eCollection 2019 Apr.</p>
			<p>PMID:
			31009447 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/30692689/" target="_blank">A
			genome-wide association study of shared risk across psychiatric
			disorders implicates gene regulation during fetal
			neurodevelopment. </a>
			</p>
			<p>Schork
			AJ, Won H, Appadurai V, Nudel R, Gandal M, Delaneau O, Revsbech
			Christiansen M, Hougaard DM, Bækved-Hansen M, Bybjerg-Grauholm J,
			Giørtz Pedersen M, Agerbo E, Bøcker Pedersen C, Neale BM, Daly
			MJ, Wray NR, Nordentoft M, Mors O, Børglum AD, Bo Mortensen P,
			Buil A, Thompson WK, Geschwind DH, Werge T.</p>
			<p>Nat
			Neurosci. 2019 Mar;22(3):353-361. doi: 10.1038/s41593-018-0320-0.
			Epub 2019 Jan 28.</p>
			<p>PMID:
			30692689 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/30305743/" target="_blank">The
			UK Biobank resource with deep phenotyping and genomic data. </a>
			</p>
			<p>Bycroft
			C, Freeman C, Petkova D, Band G, Elliott LT, Sharp K, Motyer A,
			Vukcevic D, Delaneau O, O'Connell J, Cortes A, Welsh S, Young A,
			Effingham M, McVean G, Leslie S, Allen N, Donnelly P, Marchini J.</p>
			<p>Nature.
			2018 Oct;562(7726):203-209. doi: 10.1038/s41586-018-0579-z. Epub
			2018 Oct 10.</p>
			<p>PMID:
			30305743 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/29116076/" target="_blank">The
			effect of genetic variation on promoter usage and enhancer
			activity. </a>
			</p>
			<p>Garieri
			M, Delaneau O, Santoni F, Fish RJ, Mull D, Carninci P, Dermitzakis
			ET, Antonarakis SE, Fort A.</p>
			<p>Nat
			Commun. 2017 Nov 7;8(1):1358. doi: 10.1038/s41467-017-01467-7.</p>
			<p>PMID:
			29116076 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/29058715/" target="_blank">Estimating
			the causal tissues for complex traits and diseases. </a>
			</p>
			<p>Ongen
			H, Brown AA, Delaneau O, Panousis NI, Nica AC; GTEx Consortium,
			Dermitzakis ET.</p>
			<p>Nat
			Genet. 2017 Dec;49(12):1676-1683. doi: 10.1038/ng.3981. Epub 2017
			Oct 23.</p>
			<p>PMID:
			29058715 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/29058714/" target="_blank">Predicting
			causal variants affecting expression by using whole-genome
			sequencing and RNA-seq from multiple human tissues. </a>
			</p>
			<p>Brown
			AA, Viñuela A, Delaneau O, Spector TD, Small KS, Dermitzakis ET.</p>
			<p>Nat
			Genet. 2017 Dec;49(12):1747-1751. doi: 10.1038/ng.3979. Epub 2017
			Oct 23.</p>
			<p>PMID:
			29058714 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/29022597/" target="_blank">Genetic
			effects on gene expression across human tissues. </a>
			</p>
			<p>GTEx
			Consortium; Laboratory, Data Analysis &amp;Coordinating Center
			(LDACC)—Analysis Working Group; Statistical Methods
			groups—Analysis Working Group; Enhancing GTEx (eGTEx) groups;
			NIH Common Fund; NIH/NCI; NIH/NHGRI; NIH/NIMH; NIH/NIDA;
			Biospecimen Collection Source Site—NDRI; Biospecimen Collection
			Source Site—RPCI; Biospecimen Core Resource—VARI; Brain Bank
			Repository—University of Miami Brain Endowment Bank; Leidos
			Biomedical—Project Management; ELSI Study; Genome Browser Data
			Integration &amp;Visualization—EBI; Genome Browser Data
			Integration &amp;Visualization—UCSC Genomics Institute,
			University of California Santa Cruz; Lead analysts:; Laboratory,
			Data Analysis &amp;Coordinating Center (LDACC):; NIH program
			management:; Biospecimen collection:; Pathology:; eQTL manuscript
			working group:, Battle A, Brown CD, Engelhardt BE, Montgomery SB.</p>
			<p>Nature.
			2017 Oct 11;550(7675):204-213. doi: 10.1038/nature24277.</p>
			<p>PMID:
			29022597 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/29022581/" target="_blank">The
			impact of rare variation on gene expression across tissues. </a>
			</p>
			<p>Li
			X, Kim Y, Tsang EK, Davis JR, Damani FN, Chiang C, Hess GT,
			Zappala Z, Strober BJ, Scott AJ, Li A, Ganna A, Bassik MC, Merker
			JD; GTEx Consortium; Laboratory, Data Analysis &amp;Coordinating
			Center (LDACC)—Analysis Working Group; Statistical Methods
			groups—Analysis Working Group; Enhancing GTEx (eGTEx) groups;
			NIH Common Fund; NIH/NCI; NIH/NHGRI; NIH/NIMH; NIH/NIDA;
			Biospecimen Collection Source Site—NDRI; Biospecimen Collection
			Source Site—RPCI; Biospecimen Core Resource—VARI; Brain Bank
			Repository—University of Miami Brain Endowment Bank; Leidos
			Biomedical—Project Management; ELSI Study; Genome Browser Data
			Integration &amp;Visualization—EBI; Genome Browser Data
			Integration &amp;Visualization—UCSC Genomics Institute,
			University of California Santa Cruz, Hall IM, Battle A, Montgomery
			SB.</p>
			<p>Nature.
			2017 Oct 11;550(7675):239-243. doi: 10.1038/nature24267.</p>
			<p>PMID:
			29022581 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/28516912/" target="_blank">A
			complete tool set for molecular QTL discovery and analysis. </a>
			</p>
			<p>Delaneau
			O, Ongen H, Brown AA, Fort A, Panousis NI, Dermitzakis ET.</p>
			<p>Nat
			Commun. 2017 May 18;8:15452. doi: 10.1038/ncomms15452.</p>
			<p>PMID:
			28516912 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/28186259/" target="_blank">MBV:
			a method to solve sample mislabeling and detect technical bias in
			large combined genotype and sequencing assay datasets. </a>
			</p>
			<p>Fort
			A, Panousis NI, Garieri M, Antonarakis SE, Lappalainen T,
			Dermitzakis ET, Delaneau O.</p>
			<p>Bioinformatics.
			2017 Jun 15;33(12):1895-1897. doi: 10.1093/bioinformatics/btx074.</p>
			<p>PMID:
			28186259 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/28126036/" target="_blank">Genome-wide
			analysis of differential transcriptional and epigenetic
			variability across human immune cell types. </a>
			</p>
			<p>Ecker
			S, Chen L, Pancaldi V, Bagger FO, Fernández JM, Carrillo de Santa
			Pau E, Juan D, Mann AL, Watt S, Casale FP, Sidiropoulos N, Rapin
			N, Merkel A; BLUEPRINT Consortium, Stunnenberg HG, Stegle O,
			Frontini M, Downes K, Pastinen T, Kuijpers TW, Rico D, Valencia A,
			Beck S, Soranzo N, Paul DS.</p>
			<p>Genome
			Biol. 2017 Jan 26;18(1):18. doi: 10.1186/s13059-017-1156-8.</p>
			<p>PMID:
			28126036 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/27863252/" target="_blank">The
			Allelic Landscape of Human Blood Cell Trait Variation and Links to
			Common Complex Disease. </a>
			</p>
			<p>Astle
			WJ, Elding H, Jiang T, Allen D, Ruklisa D, Mann AL, Mead D, Bouman
			H, Riveros-Mckay F, Kostadima MA, Lambourne JJ, Sivapalaratnam S,
			Downes K, Kundu K, Bomba L, Berentsen K, Bradley JR, Daugherty LC,
			Delaneau O, Freson K, Garner SF, Grassi L, Guerrero J, Haimel M,
			Janssen-Megens EM, Kaan A, Kamat M, Kim B, Mandoli A, Marchini J,
			Martens JHA, Meacham S, Megy K, O'Connell J, Petersen R, Sharifi
			N, Sheard SM, Staley JR, Tuna S, van der Ent M, Walter K, Wang SY,
			Wheeler E, Wilder SP, Iotchkova V, Moore C, Sambrook J,
			Stunnenberg HG, Di Angelantonio E, Kaptoge S, Kuijpers TW,
			Carrillo-de-Santa-Pau E, Juan D, Rico D, Valencia A, Chen L, Ge B,
			Vasquez L, Kwan T, Garrido-Martín D, Watt S, Yang Y, Guigo R,
			Beck S, Paul DS, Pastinen T, Bujold D, Bourque G, Frontini M,
			Danesh J, Roberts DJ, Ouwehand WH, Butterworth AS, Soranzo N.</p>
			<p>Cell.
			2016 Nov 17;167(5):1415-1429.e19. doi: 10.1016/j.cell.2016.10.042.</p>
			<p>PMID:
			27863252 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/27863251/" target="_blank">Genetic
			Drivers of Epigenetic and Transcriptional Variation in Human
			Immune Cells. </a>
			</p>
			<p>Chen
			L, Ge B, Casale FP, Vasquez L, Kwan T, Garrido-Martín D, Watt S,
			Yan Y, Kundu K, Ecker S, Datta A, Richardson D, Burden F, Mead D,
			Mann AL, Fernandez JM, Rowlston S, Wilder SP, Farrow S, Shao X,
			Lambourne JJ, Redensek A, Albers CA, Amstislavskiy V, Ashford S,
			Berentsen K, Bomba L, Bourque G, Bujold D, Busche S, Caron M, Chen
			SH, Cheung W, Delaneau O, Dermitzakis ET, Elding H, Colgiu I,
			Bagger FO, Flicek P, Habibi E, Iotchkova V, Janssen-Megens E, Kim
			B, Lehrach H, Lowy E, Mandoli A, Matarese F, Maurano MT, Morris
			JA, Pancaldi V, Pourfarzad F, Rehnstrom K, Rendon A, Risch T,
			Sharifi N, Simon MM, Sultan M, Valencia A, Walter K, Wang SY,
			Frontini M, Antonarakis SE, Clarke L, Yaspo ML, Beck S, Guigo R,
			Rico D, Martens JHA, Ouwehand WH, Kuijpers TW, Paul DS,
			Stunnenberg HG, Stegle O, Downes K, Pastinen T, Soranzo N.</p>
			<p>Cell.
			2016 Nov 17;167(5):1398-1414.e24. doi: 10.1016/j.cell.2016.10.026.</p>
			<p>PMID:
			27863251 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/27548312/" target="_blank">A
			reference panel of 64,976 haplotypes for genotype imputation. </a>
			</p>
			<p>McCarthy
			S, Das S, Kretzschmar W, Delaneau O, Wood AR, Teumer A, Kang HM,
			Fuchsberger C, Danecek P, Sharp K, Luo Y, Sidore C, Kwong A,
			Timpson N, Koskinen S, Vrieze S, Scott LJ, Zhang H, Mahajan A,
			Veldink J, Peters U, Pato C, van Duijn CM, Gillies CE, Gandin I,
			Mezzavilla M, Gilly A, Cocca M, Traglia M, Angius A, Barrett JC,
			Boomsma D, Branham K, Breen G, Brummett CM, Busonero F, Campbell
			H, Chan A, Chen S, Chew E, Collins FS, Corbin LJ, Smith GD,
			Dedoussis G, Dorr M, Farmaki AE, Ferrucci L, Forer L, Fraser RM,
			Gabriel S, Levy S, Groop L, Harrison T, Hattersley A, Holmen OL,
			Hveem K, Kretzler M, Lee JC, McGue M, Meitinger T, Melzer D, Min
			JL, Mohlke KL, Vincent JB, Nauck M, Nickerson D, Palotie A, Pato
			M, Pirastu N, McInnis M, Richards JB, Sala C, Salomaa V,
			Schlessinger D, Schoenherr S, Slagboom PE, Small K, Spector T,
			Stambolian D, Tuke M, Tuomilehto J, Van den Berg LH, Van Rheenen
			W, Volker U, Wijmenga C, Toniolo D, Zeggini E, Gasparini P,
			Sampson MG, Wilson JF, Frayling T, de Bakker PI, Swertz MA,
			McCarroll S, Kooperberg C, Dekker A, Altshuler D, Willer C, Iacono
			W, Ripatti S, Soranzo N, Walter K, Swaroop A, Cucca F, Anderson
			CA, Myers RM, Boehnke M, McCarthy MI, Durbin R; Haplotype
			Reference Consortium.</p>
			<p>Nat
			Genet. 2016 Oct;48(10):1279-83. doi: 10.1038/ng.3643. Epub 2016
			Aug 22.</p>
			<p>PMID:
			27548312 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/27270105/" target="_blank">Haplotype
			estimation for biobank-scale data sets. </a>
			</p>
			<p>O'Connell
			J, Sharp K, Shrine N, Wain L, Hall I, Tobin M, Zagury JF, Delaneau
			O*, Marchini J*.</p>
			<p>Nat
			Genet. 2016 Jul;48(7):817-20. doi: 10.1038/ng.3583. Epub 2016 Jun
			6.</p>
			<p>PMID:
			27270105 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/27153703/" target="_blank">Phasing
			for medical sequencing using rare variants and large haplotype
			reference panels. </a>
			</p>
			<p>Sharp
			K, Kretzschmar W, Delaneau O, Marchini J.</p>
			<p>Bioinformatics.
			2016 Jul 1;32(13):1974-80. doi: 10.1093/bioinformatics/btw065.
			Epub 2016 Feb 27.</p>
			<p>PMID:
			27153703 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26708335/" target="_blank">Fast
			and efficient QTL mapper for thousands of molecular phenotypes. </a>
			</p>
			<p>Ongen
			H, Buil A, Brown AA, Dermitzakis ET, Delaneau O.</p>
			<p>Bioinformatics.
			2016 May 15;32(10):1479-85. doi: 10.1093/bioinformatics/btv722.
			Epub 2015 Dec 26.</p>
			<p>PMID:
			26708335 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26615188/" target="_blank">The
			SIB Swiss Institute of Bioinformatics' resources: focus on curated
			databases. </a>
			</p>
			<p>SIB
			Swiss Institute of Bioinformatics Members.</p>
			<p>Nucleic
			Acids Res. 2016 Jan 4;44(D1):D27-37. doi: 10.1093/nar/gkv1310.
			Epub 2015 Nov 28.</p>
			<p>PMID:
			26615188 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" target="_blank">A
			global reference for human genetic variation. </a>
			</p>
			<p>1000
			Genomes Project Consortium, Auton A, Brooks LD, Durbin RM,
			Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean
			GA, Abecasis GR.</p>
			<p>Nature.
			2015 Oct 1;526(7571):68-74. doi: 10.1038/nature15393.</p>
			<p>PMID:
			26432245 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26423011/" target="_blank">Novel
			insights into the genetics of smoking behaviour, lung function,
			and chronic obstructive pulmonary disease (UK BiLEVE): a genetic
			association study in UK Biobank. </a>
			</p>
			<p>Wain
			LV, Shrine N, Miller S, Jackson VE, Ntalla I, Soler Artigas M,
			Billington CK, Kheirallah AK, Allen R, Cook JP, Probert K, Obeidat
			M, Bossé Y, Hao K, Postma DS, Paré PD, Ramasamy A; UK Brain
			Expression Consortium (UKBEC), Mägi R, Mihailov E, Reinmaa E,
			Melén E, O'Connell J, Frangou E, Delaneau O; OxGSK Consortium,
			Freeman C, Petkova D, McCarthy M, Sayers I, Deloukas P, Hubbard R,
			Pavord I, Hansell AL, Thomson NC, Zeggini E, Morris AP, Marchini
			J, Strachan DP, Tobin MD, Hall IP.</p>
			<p>Lancet
			Respir Med. 2015 Oct;3(10):769-81. doi:
			10.1016/S2213-2600(15)00283-0. Epub 2015 Sep 27.</p>
			<p>PMID:
			26423011 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26367535/" target="_blank">Identification
			of Genes Whose Expression Profile Is Associated with
			Non-Progression towards AIDS Using eQTLs. </a>
			</p>
			<p>Spadoni
			JL, Rucart P, Le Clerc S, van Manen D, Coulonges C, Ulveling D,
			Laville V, Labib T, Taing L, Delaneau O, Montes M, Schuitemaker H,
			Noirel J, Zagury JF.</p>
			<p>PLoS
			One. 2015 Sep 14;10(9):e0136989. doi:
			10.1371/journal.pone.0136989. eCollection 2015.</p>
			<p>PMID:
			26367535 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/26300124/" target="_blank">Population
			Variation and Genetic Control of Modular Chromatin Architecture in
			Humans. </a>
			</p>
			<p>Waszak
			SM*, Delaneau O*, Gschwind AR, Kilpinen H, Raghav SK, Witwicki RM,
			Orioli A, Wiederkehr M, Panousis NI, Yurovsky A, Romano-Palumbo L,
			Planchon A, Bielser D, Padioleau I, Udin G, Thurnheer S, Hacker D,
			Hernandez N, Reymond A, Deplancke B, Dermitzakis ET.</p>
			<p>Cell.
			2015 Aug 27;162(5):1039-50. doi: 10.1016/j.cell.2015.08.001. Epub
			2015 Aug 20.</p>
			<p>PMID:
			26300124 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/25653097/" target="_blank">Integrating
			sequence and array data to create an improved 1000 Genomes Project
			haplotype reference panel. </a>
			</p>
			<p>Delaneau
			O, Marchini J; 1000 Genomes Project Consortium; 1000 Genomes
			Project Consortium.</p>
			<p>Nat
			Commun. 2014 Jun 13;5:3934. doi: 10.1038/ncomms4934.</p>
			<p>PMID:
			25653097 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/25557783/" target="_blank">Biased
			allelic expression in human primary fibroblast single cells. </a>
			</p>
			<p>Borel
			C, Ferreira PG, Santoni F, Delaneau O, Fort A, Popadin KY, Garieri
			M, Falconnet E, Ribaux P, Guipponi M, Padioleau I, Carninci P,
			Dermitzakis ET, Antonarakis SE.</p>
			<p>Am
			J Hum Genet. 2015 Jan 8;96(1):70-80. doi:
			10.1016/j.ajhg.2014.12.001. Epub 2014 Dec 31.</p>
			<p>PMID:
			25557783 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/24980144/" target="_blank">Human
			genomic regions with exceptionally high levels of population
			differentiation identified from 911 whole-genome sequences. </a>
			</p>
			<p>Colonna
			V, Ayub Q, Chen Y, Pagani L, Luisi P, Pybus M, Garrison E, Xue Y,
			Tyler-Smith C; 1000 Genomes Project Consortium, Abecasis GR, Auton
			A, Brooks LD, DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth
			GT, McVean GA.</p>
			<p>Genome
			Biol. 2014 Jun 30;15(6):R88. doi: 10.1186/gb-2014-15-6-r88.</p>
			<p>PMID:
			24980144 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/24939907/" target="_blank">Evidence
			after imputation for a role of MICA variants in nonprogression and
			elite control of HIV type 1 infection. </a>
			</p>
			<p>Le
			Clerc S, Delaneau O, Coulonges C, Spadoni JL, Labib T, Laville V,
			Ulveling D, Noirel J, Montes M, Schächter F, Caillat-Zucman S,
			Zagury JF.</p>
			<p>J
			Infect Dis. 2014 Dec 15;210(12):1946-50. doi:
			10.1093/infdis/jiu342. Epub 2014 Jun 16.</p>
			<p>PMID:
			24939907 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/24743097/" target="_blank">A
			general approach for haplotype phasing across the full spectrum of
			relatedness. </a>
			</p>
			<p>O'Connell
			J, Gurdasani D, Delaneau O, Pirastu N, Ulivi S, Cocca M, Traglia
			M, Huang J, Huffman JE, Rudan I, McQuillan R, Fraser RM, Campbell
			H, Polasek O, Asiki G, Ekoru K, Hayward C, Wright AF, Vitart V,
			Navarro P, Zagury JF, Wilson JF, Toniolo D, Gasparini P, Soranzo
			N, Sandhu MS, Marchini J.</p>
			<p>PLoS
			Genet. 2014 Apr 17;10(4):e1004234. doi:
			10.1371/journal.pgen.1004234. eCollection 2014 Apr.</p>
			<p>PMID:
			24743097 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
			<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/24094745/" target="_blank">Haplotype
			estimation using sequencing reads. </a>
			</p>
			<p>Delaneau
			O, Howie B, Cox AJ, Zagury JF, Marchini J.</p>
			<p>Am
			J Hum Genet. 2013 Oct 3;93(4):687-96. doi:
			10.1016/j.ajhg.2013.09.002.</p>
			<p>PMID:
			24094745 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/24092746/" target="_blank">Integrative
			annotation of variants from 1092 humans: application to cancer
			genomics. </a>
			</p>
			<p>Khurana
			E, Fu Y, Colonna V, Mu XJ, Kang HM, Lappalainen T, Sboner A,
			Lochovsky L, Chen J, Harmanci A, Das J, Abyzov A, Balasubramanian
			S, Beal K, Chakravarty D, Challis D, Chen Y, Clarke D, Clarke L,
			Cunningham F, Evani US, Flicek P, Fragoza R, Garrison E, Gibbs R,
			Gümüş ZH, Herrero J, Kitabayashi N, Kong Y, Lage K, Liluashvili
			V, Lipkin SM, MacArthur DG, Marth G, Muzny D, Pers TH, Ritchie
			GRS, Rosenfeld JA, Sisu C, Wei X, Wilson M, Xue Y, Yu F; 1000
			Genomes Project Consortium, Dermitzakis ET, Yu H, Rubin MA,
			Tyler-Smith C, Gerstein M.</p>
			<p>Science.
			2013 Oct 4;342(6154):1235587. doi: 10.1126/science.1235587.</p>
			<p>PMID:
			24092746 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/23935489/" target="_blank">Association
			study of common genetic variants and HIV-1 acquisition in 6,300
			infected cases and 7,200 controls. </a>
			</p>
			<p>McLaren
			PJ, Coulonges C, Ripke S, van den Berg L, Buchbinder S, Carrington
			M, Cossarizza A, Dalmau J, Deeks SG, Delaneau O, De Luca A,
			Goedert JJ, Haas D, Herbeck JT, Kathiresan S, Kirk GD, Lambotte O,
			Luo M, Mallal S, van Manen D, Martinez-Picado J, Meyer L, Miro JM,
			Mullins JI, Obel N, O'Brien SJ, Pereyra F, Plummer FA, Poli G, Qi
			Y, Rucart P, Sandhu MS, Shea PR, Schuitemaker H, Theodorou I,
			Vannberg F, Veldink J, Walker BD, Weintrob A, Winkler CA, Wolinsky
			S, Telenti A, Goldstein DB, de Bakker PI, Zagury JF, Fellay J.</p>
			<p>PLoS
			Pathog. 2013;9(7):e1003515. doi: 10.1371/journal.ppat.1003515.
			Epub 2013 Jul 25.</p>
			<p>PMID:
			23935489 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/23269371/" target="_blank">Improved
			whole-chromosome phasing for disease and population genetic
			studies. </a>
			</p>
			<p>Delaneau
			O, Zagury JF, Marchini J.</p>
			<p>Nat
			Methods. 2013 Jan;10(1):5-6. doi: 10.1038/nmeth.2307.</p>
			<p>PMID:
			23269371 No abstract available. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/23223146/" target="_blank">A
			genome-wide association study in Caucasian women points out a
			putative role of the STXBP5L gene in facial photoaging. </a>
			</p>
			<p>Le
			Clerc S, Taing L, Ezzedine K, Latreille J, Delaneau O, Labib T,
			Coulonges C, Bernard A, Melak S, Carpentier W, Malvy D, Jdid R,
			Galan P, Hercberg S, Morizot F, Guinot C, Tschachler E, Zagury JF.</p>
			<p>J
			Invest Dermatol. 2013 Apr;133(4):929-35. doi:
			10.1038/jid.2012.458. Epub 2012 Dec 6.</p>
			<p>PMID:
			23223146 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/23128226/" target="_blank">An
			integrated map of genetic variation from 1,092 human genomes. </a>
			</p>
			<p>1000
			Genomes Project Consortium, Abecasis GR, Auton A, Brooks LD,
			DePristo MA, Durbin RM, Handsaker RE, Kang HM, Marth GT, McVean
			GA.</p>
			<p>Nature.
			2012 Nov 1;491(7422):56-65. doi: 10.1038/nature11632.</p>
			<p>PMID:
			23128226 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/22665282/" target="_blank">Haplotype
			inference. </a>
			</p>
			<p>Delaneau
			O, Zagury JF.</p>
			<p>Methods
			Mol Biol. 2012;888:177-96. doi: 10.1007/978-1-61779-870-2_11.</p>
			<p>PMID:
			22665282 
			</p>
		</td>
	</tr>
	<tr>
			<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/22362864/" target="_blank">Multicohort
			genomewide association study reveals a new signal of protection
			against HIV-1 acquisition. </a>
			</p>
			<p>Limou
			S, Delaneau O, van Manen D, An P, Sezgin E, Le Clerc S, Coulonges
			C, Troyer JL, Veldink JH, van den Berg LH, Spadoni JL, Taing L,
			Labib T, Montes M, Delfraissy JF, Schachter F, O'Brien SJ,
			Buchbinder S, van Natta ML, Jabs DA, Froguel P, Schuitemaker H,
			Winkler CA, Zagury JF.</p>
			<p>J
			Infect Dis. 2012 Apr 1;205(7):1155-62. doi: 10.1093/infdis/jis028.
			Epub 2012 Feb 23.</p>
			<p>PMID:
			22362864 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/22138821/" target="_blank">A
			linear complexity phasing method for thousands of genomes. </a>
			</p>
			<p>Delaneau
			O, Marchini J, Zagury JF.</p>
			<p>Nat
			Methods. 2011 Dec 4;9(2):179-81. doi: 10.1038/nmeth.1785.</p>
			<p>PMID:
			22138821 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/21811574/" target="_blank">Genome-wide
			association scan in HIV-1-infected individuals identifying
			variants influencing disease course. </a>
			</p>
			<p>van
			Manen D, Delaneau O, Kootstra NA, Boeser-Nunnink BD, Limou S, Bol
			SM, Burger JA, Zwinderman AH, Moerland PD, van 't Slot R, Zagury
			JF, van 't Wout AB, Schuitemaker H.</p>
			<p>PLoS
			One. 2011;6(7):e22208. doi: 10.1371/journal.pone.0022208. Epub
			2011 Jul 21.</p>
			<p>PMID:
			21811574 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/21750674/" target="_blank">CD39/adenosine
			pathway is involved in AIDS progression. </a>
			</p>
			<p>Nikolova
			M, Carriere M, Jenabian MA, Limou S, Younas M, Kök A, Huë S,
			Seddiki N, Hulin A, Delaneau O, Schuitemaker H, Herbeck JT,
			Mullins JI, Muhtarova M, Bensussan A, Zagury JF, Lelievre JD, Lévy
			Y.</p>
			<p>PLoS
			Pathog. 2011 Jul;7(7):e1002110. doi: 10.1371/journal.ppat.1002110.
			Epub 2011 Jul 7.</p>
			<p>PMID:
			21750674 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/21502085/" target="_blank">Genome-wide
			association study implicates PARD3B-based AIDS restriction. </a>
			</p>
			<p>Troyer
			JL, Nelson GW, Lautenberger JA, Chinn L, McIntosh C, Johnson RC,
			Sezgin E, Kessing B, Malasky M, Hendrickson SL, Li G, Pontius J,
			Tang M, An P, Winkler CA, Limou S, Le Clerc S, Delaneau O, Zagury
			JF, Schuitemaker H, van Manen D, Bream JH, Gomperts ED, Buchbinder
			S, Goedert JJ, Kirk GD, O'Brien SJ.</p>
			<p>J
			Infect Dis. 2011 May 15;203(10):1491-502. doi:
			10.1093/infdis/jir046.</p>
			<p>PMID:
			21502085 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/21107268/" target="_blank">Screening
			low-frequency SNPS from genome-wide association study reveals a
			new risk allele for progression to AIDS. </a>
			</p>
			<p>Le
			Clerc S, Coulonges C, Delaneau O, Van Manen D, Herbeck JT, Limou
			S, An P, Martinson JJ, Spadoni JL, Therwath A, Veldink JH, van den
			Berg LH, Taing L, Labib T, Mellak S, Montes M, Delfraissy JF,
			Schächter F, Winkler C, Froguel P, Mullins JI, Schuitemaker H,
			Zagury JF.</p>
			<p>J
			Acquir Immune Defic Syndr. 2011 Mar 1;56(3):279-84. doi:
			10.1097/QAI.0b013e318204982b.</p>
			<p>PMID:
			21107268 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/20704485/" target="_blank">Multiple-cohort
			genetic association study reveals CXCR6 as a new chemokine
			receptor involved in long-term nonprogression to AIDS. </a>
			</p>
			<p>Limou
			S, Coulonges C, Herbeck JT, van Manen D, An P, Le Clerc S,
			Delaneau O, Diop G, Taing L, Montes M, van't Wout AB, Gottlieb GS,
			Therwath A, Rouzioux C, Delfraissy JF, Lelièvre JD, Lévy Y,
			Hercberg S, Dina C, Phair J, Donfield S, Goedert JJ, Buchbinder S,
			Estaquier J, Schächter F, Gut I, Froguel P, Mullins JI,
			Schuitemaker H, Winkler C, Zagury JF.</p>
			<p>J
			Infect Dis. 2010 Sep 15;202(6):908-15. doi: 10.1086/655782.</p>
			<p>PMID:
			20704485 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/19754311/" target="_blank">Genomewide
			association study of a rapid progression cohort identifies new
			susceptibility alleles for AIDS (ANRS Genomewide Association Study
			03). </a>
			</p>
			<p>Le
			Clerc S, Limou S, Coulonges C, Carpentier W, Dina C, Taing L,
			Delaneau O, Labib T, Sladek R; ANRS Genomic Group, Deveau C,
			Guillemain H, Ratsimandresy R, Montes M, Spadoni JL, Therwath A,
			Schächter F, Matsuda F, Gut I, Lelièvre JD, Lévy Y, Froguel P,
			Delfraissy JF, Hercberg S, Zagury JF.</p>
			<p>J
			Infect Dis. 2009 Oct 15;200(8):1194-201. doi: 10.1086/605892.</p>
			<p>PMID:
			19754311 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/19115949/" target="_blank">Genomewide
			association study of an AIDS-nonprogression cohort emphasizes the
			role played by HLA genes (ANRS Genomewide Association Study 02). </a>
			</p>
			<p>Limou
			S, Le Clerc S, Coulonges C, Carpentier W, Dina C, Delaneau O,
			Labib T, Taing L, Sladek R, Deveau C, Ratsimandresy R, Montes M,
			Spadoni JL, Lelièvre JD, Lévy Y, Therwath A, Schächter F,
			Matsuda F, Gut I, Froguel P, Delfraissy JF, Hercberg S, Zagury JF;
			ANRS Genomic Group.</p>
			<p>J
			Infect Dis. 2009 Feb 1;199(3):419-26. doi: 10.1086/596067.</p>
			<p>PMID:
			19115949 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/19087329/" target="_blank">Shape-IT:
			new rapid and accurate algorithm for haplotype inference. </a>
			</p>
			<p>Delaneau
			O, Coulonges C, Zagury JF.</p>
			<p>BMC
			Bioinformatics. 2008 Dec 16;9:540. doi: 10.1186/1471-2105-9-540.</p>
			<p>PMID:
			19087329 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/17573965/" target="_blank">ISHAPE:
			new rapid and accurate software for haplotyping. </a>
			</p>
			<p>Delaneau
			O, Coulonges C, Boelle PY, Nelson G, Spadoni JL, Zagury JF.</p>
			<p>BMC
			Bioinformatics. 2007 Jun 15;8:205. doi: 10.1186/1471-2105-8-205.</p>
			<p>PMID:
			17573965 Free PMC article. 
			</p>
		</td>
	</tr>
	<tr>
		<td><p><a href="https://pubmed.ncbi.nlm.nih.gov/17067372/" target="_blank">Computation
			of haplotypes on SNPs subsets: advantage of the &quot;global
			method&quot;. </a>
			</p>
			<p>Coulonges
			C, Delaneau O, Girard M, Do H, Adkins R, Spadoni JL, Zagury JF.</p>
			<p>BMC
			Genet. 2006 Oct 26;7:50. doi: 10.1186/1471-2156-7-50.</p>
			<p>PMID:
			17067372 Free PMC article. 
			</p>
		</td>
	</tr>
</table>


<!-- End of PUBLICATION vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->


            </div>
        </section><!-- End of portfolio-one Section -->

        <!-- Sections -->
        <section id="contact" class="contact">
		<div class="container">
                	<div class="row">
				<div class="col-md-4 col-sm-4 col-xs-12">
					<div class="contact-left-info text-center">
        			                <h5>Find Us</h5>
        			                <div class="separator2"></div>
        			        	<p>University of Lausanne</p>
						<p>Department of Computational Biology</p>
						<p>Genopode building, Bureau GEN-2015</p>
						<p>Quarter UNIL-Sorge, 1015 Lausanne</p>
						<p>Switzerland</p>
					</div>
				</div> 

	       			<div class="col-md-4 col-sm-4 col-xs-12">
        				<div class="contact-left-info text-center">
        		                	<h5>Call Us</h5>
		                                <div class="separator2"></div>
                			        <p>Direct line: +41 (0)21 692 40 55</p>
						<p>Secretary: +41 (0)21 692 53 97</p>
                			</div>
				</div>

				<div class="col-md-4 col-sm-4 col-xs-12">
					<div class="contact-left-info text-center">
						<h5>Email Us</h5>
						<div class="separator2"></div>
                				<p>olivier.delaneau@unil.ch</p>
					</div>
				</div>
			</div>
		</div> <!-- /container -->       
        </section>

        <!--Footer-->
        <footer id="footer" class="footer">
            <div class="container">

                <div class="scroll-top">
		<a href="#home">
                    <div class="scrollup">
                        <i class="fa fa-angle-up"></i>
                    </div>
		</a>

                </div>
                <div class="row">
                    <div class="col-md-6 col-sm-6 col-xs-12">
                        <div class="copyright">
                            <p>&copy; 2022 All Rights Reserved / Modified theme from <a href="http://bootstrapthemes.co">http://bootstrapthemes.co</a></p>
                        </div>
                    </div>

                    <div class="col-md-6 col-sm-6 col-xs-12">
                        <div class="social text-right">
                            <a target="_blank" href="https://twitter.com/delaneaugroup"><i class="fa fa-twitter"></i></a>
                            <a target="_blank" href="https://www.linkedin.com/in/odelaneau/"><i class="fa fa-linkedin"></i></a>
                        </div>
                    </div>
                </div>
            </div>
        </footer>


        <script src="assets/js/vendor/bootstrap.min.js"></script>
        <script src="assets/js/vendor/isotope.min.js"></script>
        <script src="assets/js/plugins.js"></script>
         <script src="assets/js/jquery.magnific-popup.js"></script>
        <script src="assets/js/main.js"></script>
    </body>
</html>
